.github/workflows/test.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         # Python 3.9 does not run on macos-latest (14)
         # Uses macos-13 for 3.9 and macos-latest for >=3.10 
         os: [ubuntu-latest, macos-13, macos-latest, windows-latest]
+        numpy: ["numpy-v1", "numpy-v2"]
```

Comment:
Before we merge, we should check that the tests pass on all OSs. What I normally do for that is make a commit where this CI job runs on pull request and if everything passes I then revert it